state/execution.go

```diff
@@ -243,6 +243,12 @@ func (blockExec *BlockExecutor) ApplyBlock(
 		return state, 0, ErrProxyAppConn(err)
 	}
 
+	for _, tx := range abciResponses.DeliverTxs {
```

Comment:
this could be interesting! since this only shows us failed txs, I think we should name the metric as such. Rejected txs means something else than failed imo.

To close rejected txs, we actually care about how many transactions are rejected from prepare proposal not from deliver

we are actually most interesting in how many txs from process proposal are rejected

Reply:
I think failed txs mean rejected txs by application in a block

In `cat`, failed tx also means rejected tx

Reply:
The distinction is that "rejected" txs never made it in the block. `CheckTx` and `PrepareProposal` thus "reject" transactions. "failed" means the transaction was included and committed in a block but failed to execute